### ISSUE TITLE
Replicate official models cannot be used as of promptfoo 0.61

### ISSUE THREAD

**Issue by aaronstillwell** [promptfoo/promptfoo#1058]
I have been using promptfoo to test prompts against official models provided by Replicate, such as [meta/meta-llama-3-8b-instruct](https://replicate.com/meta/meta-llama-3-8b-instruct).

I attempted to run an eval today for the first time in a while and was greeted with the following error in my evaluation output after updating promptfoo to the latest version.

`[FAIL] API call error: Error: Invalid reference to model version: meta/meta-llama-3-8b-instruct:. Expected format: owner/name or owner/name:version`

I found this strange, as I use the owner/name format in my promptfoo config and have not changed it.

Here is a snippet from the config:

```yaml
  - id: replicate:meta/meta-llama-3-8b-instruct
    config:
      temperature: 0
      # continued...
```

I subsquently sought to specify a version for the model, but the [replicate API](https://replicate.com/docs/reference/http#models.versions.list) returns the following for this model:
`{"status":404,"title":"Not found","detail":"This model is an official model and does not expose a list of versions."}`

After manually walking back the promptfoo version, I'm seeing my setup work fine in `0.60` but not `0.61`. I do not see anything explicit in the changelog for `0.61` that would cause this.

**Comment by typpo** [comment]
Thanks for flagging, @aaronstillwell!  #1059 fixes this issue.

**Comment by mldangelo** [comment]
A fix for this was deployed in [0.68.1](https://github.com/promptfoo/promptfoo/releases/tag/0.68.1) Please let us know if you run into any other issues and thanks again for reporting it!
